processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByQueryEngineV2.java

```diff
@@ -159,57 +160,29 @@ public GroupByEngineIterator make()
                             final ByteBuffer buffer = bufferHolder.get();
```

Comment:
I already tried that one, but two sequences work differently. Originally the bufferHolder is released after the inner MappedSequence is accumulated, but it is released after the ConcatSequence is accumulated.